apps/frontend/src/app/pages/maps/map-info/map-info.component.html

```diff
@@ -34,7 +34,7 @@
           <m-icon class="mr-2 pb-0.5" icon="play-circle" />
           Play
         </button>
-        @if (map.downloadURL ?? map.submission?.currentVersion?.downloadURL; as bsp) {
+        @if (map.currentVersion; as bsp) {
```

Comment:
Either this should be `map.currentVersion.downloadURL`, or you should add `.downloadURL` on line 46

Reply:
Good spot, fixed